flow-nodes/api-builder-plugin-petstore/README.md

```diff
@@ -0,0 +1,15 @@
+# api-builder-plugin-petstore
+An API Builder flow-node plugin that exports Petstore API functions.
+
+Internally, the plugin uses the same engine as `@axway/api-builder-plugin-fn-swagger` to package and version OAS 2/3 specs from the plugin's `openapi` directory in a managed way.
```

Comment:
Maybe. as i said above, nobody really cares about the OAS version as long as they have a working flow-node. the plugin could have implemented the petstore manually without any OAS. Although this Internal stuff and other parts of the readme are specifically for the person picking up the template and seeing how it works. Even in that case, they shouldn't care too much about the OAS version, just that they can drop any OAS file into it and it will work. 

The info about the config file can probably go here for both types of users though

Reply:
Get rid of the bits about "same engine" and swagger-plugin.  Keep the bit about OAS 2/3 specs from `openapi`.